lib/common/iso8601.c

```diff
@@ -615,20 +611,17 @@ crm_time_parse(const char *time_str, crm_time_t * a_time)
     if (time_str) {
         dt->seconds = crm_time_parse_sec(time_str);
 
-        offset_s = strstr(time_str, "Z");
-        if (offset_s == NULL) {
-            offset_s = strstr(time_str, " ");
-        }
+        /* standard doesn't allow spaces, but be tolerant */
+        offset_s = time_str + strcspn(time_str, " -+Z");
     }
 
-    if (offset_s) {
-        while (isspace(offset_s[0])) {
-            offset_s++;
-        }
+    while (isspace(*offset_s)) {
```

Comment:
offset_s will be NULL if time_str is NULL, so this still needs to be guarded (or moved into the "if (time_str)" block)

Reply:
indeed, thanks, not sure if overlook or feeling that I've done something
that was in the plan but got stuck in the queue in practice :-)